packs/st2cd/actions/workflows/st2_pkg_ubuntu14.yaml

```diff
@@ -161,5 +161,13 @@
       params:
         hosts: "{{build_server}}"
         cmd: "rm -f /home/stanley/*{{version}}-{{build}}*.deb"
+      on-success: "push_st2client_to_pypi"
```

Comment:
Why not break this out and do it as a rule? This keeps the build pipeline more flexible.